### PR TITLE
fix the validation errors

### PR DESCRIPTION
**Pull request by ericshape**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

Fix the validation errors and release the 2020-11-01 preview version. There is no API contract change for the PrivateEndpointConnect specs. Thus, we are OK to use Microsoft.Sql/preview/2020-11-01-preview/PrivateEndpointConnections.json to replace the 2018-06-01 one. 

The goal of 2020-11-01 is to have a Backup Restore feature in .NET SDK and Powershell. cc @xaliciayang 
And we will not release any Python / Go SDK. Thus, the known "import-export" namespace issue will not be a problem for .NET SDK and Powershell. 

We will fix "import/export" issue in the upcoming V5 SDK. 

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [x] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ericshape** [comment]
All the .NET test errors will  be fixed by @xaliciayang in .NET SDK repo.

**Review by bcham — APPROVED** [review]

File: specification/sql/resource-manager/readme.md [hunk 1]
```diff
@@ -127,6 +126,7 @@ input-file:
 - Microsoft.Sql/preview/2020-11-01-preview/Databases_legacy.json 
 - Microsoft.Sql/preview/2020-11-01-preview/LongTermRetentionBackups.json
 - Microsoft.Sql/preview/2020-11-01-preview/LongTermRetentionPolicies.json
+- Microsoft.Sql/preview/2020-11-01-preview/PrivateEndpointConnections.json
```
Comment (akning-ms) [inline specification/sql/resource-manager/readme.md:129]: not sure what validation error will be fixed.  is there any different between 2 privatelintendpointconnection?

Comment (ericshape) [reply specification/sql/resource-manager/readme.md:129]: without my PR , the validation errors are:
https://github.com/Azure/azure-rest-api-specs/pull/13321/checks?check_run_id=2060056651

I used the 2020-11-01 private connection spec to replace the 2018-06-01 one. I look through it and did not find any API contract change. The only change is private connection state is switched from string to enum string.

Comment (ericshape) [reply specification/sql/resource-manager/readme.md:129]: the spec files differences are: https://www.diffchecker.com/bFMuIpaE
From my perspective, they are the same.

**Pull request merged**